besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3963,4 +3972,31 @@ public void quorumInteropEnabledSucceedsWithGasPriceSetToZero() throws IOExcepti
         "0");
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
+
+  @Test
+  public void quorumInteropEnabledFailsWithMainnetDefaultNetwork() throws IOException {
+    final Path genesisFile = createFakeGenesisFile(INVALID_GENESIS_QUORUM_INTEROP_ENABLED_MAINNET);
+    parseCommand(
+        "--goquorum-compatibility-enabled",
+        "--genesis-file",
+        genesisFile.toString(),
+        "--min-gas-price",
+        "0");
+    assertThat(commandErrorOutput.toString())
+        .contains("GoQuorum compatibility mode (enabled) cannot be used on Mainnet");
+  }
+
+  @Test
+  public void quorumInteropEnabledFailsWithMainnetChainId() throws IOException {
+    final Path genesisFile =
+        createFakeGenesisFile(INVALID_GENESIS_QUORUM_INTEROP_ENABLED_MAINNET.put("chainId", "1"));
```

Comment:
Could you use the valid quorum genesis file here with chainId 1?

Reply:
yes that would work

Reply:
but it's used in two spots (and the other one doesn't add chainId) so I think it's clearer to use the constant